co-authors-plus.php

```diff
@@ -380,14 +383,15 @@ public function coauthors_meta_box( $post ) {
 				<?php
 				foreach ( $coauthors as $coauthor ) :
 					$count++;
+					$avatar_url = get_avatar_url( $coauthor->ID, array( 'default' => 'gravatar_default' ) );
 					?>
 					<li>
-						<?php echo get_avatar( $coauthor->user_email, $this->gravatar_size ); ?>
```

Comment:
Since we're filtering `get_avatar_url`, we should still be able to use `get_avatar`.